recipes/fastnanquantile/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "fastnanquantile" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1d0059370a3e00f706531ed2f390433ea3d0d3dadc31574fd2c2cfde440ee326
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
```

Comment:
See the linter message, you probably need setuptools here.

Reply:
Added.